2018-03-22T15:05:10.032Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Failed to add object with ID '002' as the object already exists)
    at /home/saif/.nvm/versions/node/v8.9.4/lib/node_modules/composer-rest-server/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-03-22T15:05:10.033Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID '002' as the object already exists)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID '002' as the object already exists)\n\n    at _initializeChannel.then.then.then.then.catch (/home/saif/.nvm/versions/node/v8.9.4/lib/node_modules/composer-rest-server/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-03-22T15:05:29.394Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Failed to add object with ID '002' as the object already exists)
    at /home/saif/.nvm/versions/node/v8.9.4/lib/node_modules/composer-rest-server/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-03-22T15:05:29.394Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID '002' as the object already exists)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID '002' as the object already exists)\n\n    at _initializeChannel.then.then.then.then.catch (/home/saif/.nvm/versions/node/v8.9.4/lib/node_modules/composer-rest-server/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
